packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -276,20 +276,24 @@ There are number of new environment variables, and you may use this table as a r
 
 ### Create Accounts from the `ReleaseGold` contracts
 
-In order to use the balances from `ReleaseGold` contracts, we need to create associated Accounts. In the Baklava network, you can look up your Beneficiary address in [the published mapping](https://gist.githubusercontent.com/nategraf/a87f9c2e488ab2d38a0a3c09f5d4ca2b/raw) to find your `ReleaseGold` contract addresses. If you are a genesis validator, your two Beneficary addresses will be the provided `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`.
+In order to participate on the network (lock gold, vote, validate) from a `ReleaseGold` contract, we need to create an Account at the address of the `ReleaseGold` contract. In the Baklava network, you can look up your Beneficiary address in [the published mapping](https://gist.githubusercontent.com/nategraf/a87f9c2e488ab2d38a0a3c09f5d4ca2b/raw) to find your `ReleaseGold` contract addresses. If you are a genesis validator, your two Beneficary addresses will be the provided `CELO_VALIDATOR_BENEFICIARY_ADDRESS` and `CELO_VALIDATOR_GROUP_BENEFICIARY_ADDRESS`.
+
+{% hint style="info" %}
+An earlier version of this document set the beneficiary addresses to `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`, but the attestation service expects these to refer to the `ReleaseGold` contract, so please move your beneficiary addresses to reflect the environment variables above.
+{% endhint %}
 
 ```bash
 # On your local machine
-export CELO_VALIDATOR_GROUP_RG_ADDRESS=<YOUR-CELO-VALIDATOR-GROUP-RG-ADDRESS>
-export CELO_VALIDATOR_RG_ADDRESS=<YOUR-CELO-VALIDATOR-RG-ADDRESS>
+export CELO_VALIDATOR_GROUP_ADDRESS=<YOUR-CELO-VALIDATOR-GROUP-RG-ADDRESS>
```

Comment:
Why are we changing the env var names here?

Reply:
For the sake of the attestation service. The service expects `CELO_VALIDATOR_ADDRESS` to be the validator account address, and this is the more correct version as well.
Currently `CELO_VALIDATOR_ADDRESS` in this doc refers to the beneficiary which will cause issues for attestation (confusing it for the validator) so we decided to change it rather than rework parts of attestation